src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json

```diff
@@ -2659,8 +2659,8 @@
       }
     }
   },
-  "illustrated-message-maximum-width": {
-    "value": "380px",
+  "🚫 illustrated-message-maximum-width": {
+    "value": "{illustrated-message-vertical-maximum-width}",
```

Comment:
@PaliwalSparsh @lynnhao @GarthDB @mrcjhicks 
This is an example of a working deprecation. the new token `illustrated-message-vertical-maximum-width` has the same value `380px`. This is okay to do. 👍